src/Parser/Shaman/Restoration/Constants.js

```diff
@@ -31,7 +31,7 @@ export const ABILITIES_AFFECTED_BY_MASTERY = [
   SPELLS.HEALING_RAIN_HEAL.id,
   SPELLS.UNLEASH_LIFE_TALENT.id,
   SPELLS.EARTH_SHIELD_HEAL.id,
-  //SPELLS.DOWNPOUR_TALENT.id, Not yet, but expected to get fixed
+  SPELLS.DOWNPOUR_TALENT.id,
   SPELLS.DOWNPOUR.id,
```

Comment:
Is it intended that there are both `SPELLS.DOWNPOUR_TALENT.id` and `SPELLS.DOWNPOUR.id` here?

Reply:
yes, Downpour_Talent is the newly added talent which is basically the artifact active, Downpour is T21 4 piece. i have no clue why they have the same name